lib/api/users/user.ts

```diff
@@ -0,0 +1,27 @@
+import { db} from "../../db";
+
+export function getlockers() {
+    return db.prepare("SELECT * FROM lockers").get()
+}
+
+export function register(building: number, number: number, name: string) {
+
+    const user_statement = db.prepare("INSERT INTO users VALUES(?)").run(name)
```

Comment:
Remember that a user has both a name and an email

Reply:
Good one to write a test case for.